tests/acceptance/invoices-test.js

```diff
@@ -66,6 +66,57 @@ test('print invoice', function(assert) {
   });
 });
 
+//test pricing profile
```

Comment:
@tangollama your code is throwing JSCS (formatting) errors on test.  You can verify these yourself by running ember test: 
```
  acceptance/invoices-test.js should pass jscs.
            requireSpaceAfterLineComment: Missing space after line comment at acceptance/invoices-test.js :
                67 |});
                68 |
                69 |//test pricing profile
            -------------------^
                70 |test('pricing profiles', function(assert) {
                71 |  runWithPouchDump('billing', function() {
            requireSpaceAfterLineComment: Missing space after line comment at acceptance/invoices-test.js :
                77 |      waitToAppear('h4:contains(New Pricing Profile)');
                78 |    });
                79 |    //% discount
            ------------------^
                80 |    andThen(function() {
                81 |      fillIn('.pricing-profile-name input', '50% profile');
            disallowTrailingWhitespace: Illegal trailing whitespace at acceptance/invoices-test.js :
                80 |    andThen(function() {
                81 |      fillIn('.pricing-profile-name input', '50% profile');
                82 |      fillIn('.pricing-profile-percentage input', '50');      
            ----------------------------------------------------------------^
                83 |      click('button:contains(Add)');
                84 |      waitToAppear('button:contains(Ok)');
            requireSpacesInFunction: Missing space before opening curly brace at acceptance/invoices-test.js :
                85 |      click('button:contains(Ok)');
                86 |    });
                87 |    andThen(function(){
            ------------------------------^
                88 |      click('button:contains(+ new item)');
                89 |      waitToAppear('h4:contains(New Pricing Profile)');
            requireSpaceBeforeBlockStatements: One (or more) spaces required before opening brace for block expressions at acceptance/invoices-test.js :
                85 |      click('button:contains(Ok)');
                86 |    });
                87 |    andThen(function(){
            ------------------------------^
                88 |      click('button:contains(+ new item)');
                89 |      waitToAppear('h4:contains(New Pricing Profile)');
            requireSpaceAfterLineComment: Missing space after line comment at acceptance/invoices-test.js :
                89 |      waitToAppear('h4:contains(New Pricing Profile)');
                90 |    });
                91 |    //flat discount
            --------------------^
                92 |    andThen(function() {
                93 |      fillIn('.pricing-profile-name input', '$100 discount');
            disallowTrailingWhitespace: Illegal trailing whitespace at acceptance/invoices-test.js :
                92 |    andThen(function() {
                93 |      fillIn('.pricing-profile-name input', '$100 discount');
                94 |      fillIn('.pricing-profile-discount input', '100');      
            ---------------------------------------------------------------^
                95 |      click('button:contains(Add)');
                96 |      waitToAppear('button:contains(Ok)');
            requireSpaceBeforeBlockStatements: One (or more) spaces required before opening brace for block expressions at acceptance/invoices-test.js :
                97 |      click('button:contains(Ok)');
                98 |    });
                99 |    andThen(function(){
            ------------------------------^
               100 |      click('button:contains(+ new item)');
               101 |      waitToAppear('h4:contains(New Pricing Profile)');
            requireSpacesInFunction: Missing space before opening curly brace at acceptance/invoices-test.js :
                97 |      click('button:contains(Ok)');
                98 |    });
                99 |    andThen(function(){
            ------------------------------^
               100 |      click('button:contains(+ new item)');
               101 |      waitToAppear('h4:contains(New Pricing Profile)');
            requireSpaceAfterLineComment: Missing space after line comment at acceptance/invoices-test.js :
               101 |      waitToAppear('h4:contains(New Pricing Profile)');
               102 |    });
               103 |    //flat fee
            -----------------^
               104 |    andThen(function() {
               105 |      fillIn('.pricing-profile-name input', '$150 fee');
            disallowTrailingWhitespace: Illegal trailing whitespace at acceptance/invoices-test.js :
               104 |    andThen(function() {
               105 |      fillIn('.pricing-profile-name input', '$150 fee');
               106 |      fillIn('.pricing-set-fee input', '150');      
            ------------------------------------------------------^
               107 |      click('button:contains(Add)');
               108 |      waitToAppear('button:contains(Ok)');
        Log: |
```